code/go/0chain.net/smartcontract/storagesc/sc.go

```diff
@@ -151,7 +151,8 @@ func (ssc *StorageSmartContract) setSC(sc *sci.SmartContract, _ sci.BCContextI)
 	ssc.SmartContract.RestHandlers["/alloc_read_size"] = ssc.GetReadAmountHandler
 	ssc.SmartContract.RestHandlers["/alloc_write_marker_count"] = ssc.GetWriteMarkerCountHandler
 	ssc.SmartContract.RestHandlers["/errors"] = ssc.GetErrors
-	// blobber aggregated saved data
+	ssc.SmartContract.RestHandlers["/get_alloc_blobbers"] = ssc.GetAllocationBlobbersHandler
```

Comment:
Maybe `/alloc_blobbers` as the Rest http GET method already has the get action. So as the `free_alloc_blobbers` below.